{"name": "test_user_auth[case20]", "status": "failed", "statusDetails": {"message": "AssertionError: 状态码不符：预期200，实际403，完整response：<Response [403]>\nassert 403 == 200\n +  where 403 = <Response [403]>.status_code", "trace": "case = {'case_id': 'auth_021', 'expected_code': 200, 'expected_message': '个人信息更新成功', 'expected_success': True, ...}\nauth_token = 'eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJhZG1pbjExNTUiLCJpYXQiOjE3NTM5NzM2MTksImV4cCI6MTc1Mzk4MDgxOX0.o6NEEm1X7wcAxCVuV-IINMTTJ9O8Xd9NVTOfxEXpIF4'\nsmall_jpg_image = 'small_4647.jpg', large_jpg_image = 'large_5110.jpg'\nsmall_png_image = 'small_4593.png', large_png_image = 'large_1148.png'\nnon_image_file = 'test_3161.txt'\n\n    @allure.feature(\"用户认证模块\")\n    @pytest.mark.parametrize(\"case\", CASES)\n    def test_user_auth(case, auth_token, small_jpg_image, large_jpg_image, small_png_image, large_png_image, non_image_file):\n        allure.story(case[\"name\"])\n        allure.title(f\"{case['case_id']}: {case['name']}\")\n    \n        # 文件映射：用例文件名 -> 实际测试文件\n        file_map = {\n            \"test_avatar.jpg\": small_jpg_image,\n            \"large_avatar.jpg\": large_jpg_image,\n            \"test_avatar.png\": small_png_image,\n            \"large_avatar.png\": large_png_image,\n            \"test_file.txt\": non_image_file\n        }\n    \n        # 预处理请求\n        request_data = preprocess_request(case, auth_token, file_map)\n    \n        # 注册用例单独处理验证码\n        if (case[\"case_id\"].startswith(\"auth_00\")\n            and case[\"method\"] == \"POST\"\n            and case[\"url\"] == \"/auth/register\"):\n    \n            # 获取注册邮箱\n            email = request_data[\"json\"][\"email\"]\n            # 发送并获取验证码\n            verify_code = send_and_get_verify_code(email)\n            # 填充验证码到请求参数\n            request_data[\"json\"][\"verifyCode\"] = verify_code\n    \n        # 发送请求\n        response = Request.send_request(\n            method=case[\"method\"],\n            endpoint=case[\"url\"],\n            json=request_data.get(\"json\"),\n            params=request_data.get(\"params\"),\n            headers=request_data.get(\"headers\"),\n            files=request_data.get(\"files\"),\n            token=auth_token if case[\"case_id\"] != \"auth_029\" else None  # 未登录用例移除token\n        )\n    \n        # 断言\n>       assert_response(\n            response,\n            expected_code=case[\"expected_code\"],\n            expected_success=case.get(\"expected_success\"),\n            expected_message=case.get(\"expected_message\")\n        )\n\ntests\\test_user_auth.py:71: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nresponse = <Response [403]>, expected_code = 200, expected_success = True\nexpected_message = '个人信息更新成功'\n\n    @allure.step(\"验证响应：状态码{expected_code}，消息{expected_message}\")\n    def assert_response(response, expected_code, expected_success=None, expected_message=None):\n        \"\"\"验证响应结果\"\"\"\n>       assert response.status_code == expected_code, \\\n            f\"状态码不符：预期{expected_code}，实际{response.status_code}，完整response：{response}\"\nE       AssertionError: 状态码不符：预期200，实际403，完整response：<Response [403]>\nE       assert 403 == 200\nE        +  where 403 = <Response [403]>.status_code\n\nutils\\test_utils.py:7: AssertionError"}, "steps": [{"name": "发送请求：'PUT' '/auth/userinfo'", "status": "passed", "attachments": [{"name": "请求信息", "source": "90cb4791-09f9-48f7-8613-a689e8a4b941-attachment.json", "type": "application/json"}, {"name": "响应信息", "source": "ceed5da1-83ba-4751-ad55-73f7a09c2b0b-attachment.txt", "type": "text/plain"}, {"name": "响应体", "source": "99024c39-7799-4ca2-8d29-dcefd165b2dc-attachment.json", "type": "application/json"}], "parameters": [{"name": "method", "value": "'PUT'"}, {"name": "endpoint", "value": "'/auth/userinfo'"}, {"name": "headers", "value": "{'Authorization': 'Bearer '}"}, {"name": "params", "value": "{}"}, {"name": "json", "value": "{'oldPassword': 'admin1155', 'newPassword': 'NewTest789', 'securityEmail': 'admin1155@example.com', 'verificationCode': '{verify_code}'}"}, {"name": "files", "value": "None"}, {"name": "token", "value": "'eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJhZG1pbjExNTUiLCJpYXQiOjE3NTM5NzM2MTksImV4cCI6MTc1Mzk4MDgxOX0.o6NEEm1X7wcAxCVuV-IINMTTJ9O8Xd9NVTOfxEXpIF4'"}], "start": 1753973668311, "stop": 1753973668339}, {"name": "验证响应：状态码200，消息'个人信息更新成功'", "status": "failed", "statusDetails": {"message": "AssertionError: 状态码不符：预期200，实际403，完整response：<Response [403]>\nassert 403 == 200\n +  where 403 = <Response [403]>.status_code\n", "trace": "  File \"D:\\Anaconda3\\Lib\\site-packages\\allure_commons\\_allure.py\", line 202, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"D:\\工作记录\\项目说明\\api_test\\utils\\test_utils.py\", line 7, in assert_response\n    assert response.status_code == expected_code, \\\n"}, "parameters": [{"name": "response", "value": "<Response [403]>"}, {"name": "expected_code", "value": "200"}, {"name": "expected_success", "value": "True"}, {"name": "expected_message", "value": "'个人信息更新成功'"}], "start": 1753973668339, "stop": 1753973668339}], "parameters": [{"name": "case", "value": "{'case_id': 'auth_021', 'name': '旧密码正确+新密码符合强度要求', 'method': 'PUT', 'url': '/auth/userinfo', 'headers': {'Authorization': 'Bearer '}, 'json': {'oldPassword': 'admin1155', 'newPassword': 'NewTest789', 'securityEmail': 'admin1155@example.com', 'verificationCode': '{verify_code}'}, 'expected_code': 200, 'expected_success': True, 'expected_message': '个人信息更新成功'}"}], "start": 1753973668311, "stop": 1753973668340, "uuid": "c552660a-0a29-4df4-bed0-dd5c1dba0701", "historyId": "24f13640df5a7716fa28cd3732c78f43", "testCaseId": "de18b51a986ebd96b4485f2c6341275e", "fullName": "tests.test_user_auth#test_user_auth", "labels": [{"name": "feature", "value": "用户认证模块"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_auth"}, {"name": "host", "value": "CYJ"}, {"name": "thread", "value": "5344-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_auth"}], "titlePath": ["tests", "test_user_auth.py"]}